superset/models/core.py

```diff
@@ -556,13 +556,17 @@ def get_all_table_names_in_schema(  # pylint: disable=unused-argument
         :param cache: whether cache is enabled for the function
         :param cache_timeout: timeout in seconds for the cache
         :param force: whether to force refresh the cache
-        :return: list of tables
+        :return: set of tables
```

Comment:
code style: Some people consider including the type in the return description bad practice if it's already typed in the sig. I'm not sure which camp I belong to (it varies from day to day), but maybe we could rather try to be more expressive regarding the funny `Tuple[str, str]` thingy, something like
```
:return: table name and schema pairs
```